src/rt_redbug.erl

```diff
@@ -26,21 +26,22 @@
 -module(rt_redbug).
 
 -export([is_tracing_applied/0]).
--export([new/0]).
 -export([set_tracing_applied/1]).
 -export([stop/1]).
 -export([trace/2, trace/3]).
 
-%% Create a new ets to store globally whether we're tracing or not.
-new() -> 
-    ok.
-
 set_tracing_applied(TracingApplied) when is_boolean(TracingApplied) ->
-    lager:info("Setting apply tracing to ~p", [TracingApplied]),
-    rt_config:set(apply_traces, TracingApplied).
+    %% If this flag is set on the command line, but not in the config file
+    %% then switch it on.
+    Enabled = case rt_config:get(apply_traces, undefined) of
```

Comment:
Doesn't this allow the config (when `false`) to override the command line? I think you want to use `false` instead of `undefined` as your default value and first case, but haven't thought it entirely through.